sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -47,14 +47,6 @@ It supports .Net Core on every platform (Windows, macOS, Linux).
 | project you want to analyze has been built with. Concretely, you can analyze .NET Core code with the .NET Framework version of 
 | the Scanner. It's only relevant depending on your OS, and on the versions of .NET SDKs that are installed on your build machine.
 
-<!-- sonarqube -->
-### Compatibility
-
-Scanner Version|SonarQube
----|---
-5.x| LTS 6.7+
```

Comment:
Duncan's message says `We should drop this section and make it clear which versions of the scanner are supported.`

So maybe we should mention we support the latest version of the scanner? @tom-howlett-sonarsource ?

But let's drop close this PR and open a new one for that topic.

Reply:
I chatted to @tom-howlett-sonarsource about this offline, the conclusion was:
_Yes, the table should go, and I don't think we need to add anything on versions supported, we try not to break old versions and AKAIK we don't have any compatibility issues._